src/Calculator/Views/GraphingCalculator/KeyGraphFeaturesPanel.xaml

```diff
@@ -12,96 +12,12 @@
              mc:Ignorable="d">
```

Comment:
In dark mode all of the text in the MathRichEdit boxes on this page are now black instead of white, making them very hard to read.